### PR TITLE
Branch 1

### PR DESCRIPTION
**Pull request by javdad**
Created new project. HW1-2-3 Included. Could't split them into different PR's, honestly don't know how. Still working on and will add more code. Hope for your feedback.

### CODE REVIEW

File: models/account.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+	},
+	balance: {
+		type: Number,
+		default: 0,
+	},
+	dateOfCreation: {
+		type: Date,
+		default: Date.now,
+	},
+	user: {
```
Comment (AudriusBaltusis) [inline models/account.js]: `userId`

File: controllers/categoriesControl.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+		try {
+			res.send("Get category");
+		} catch (err) {
+			res.send(err);
+		}
+	}
+
+	createCategory(req, res) {
+		try {
+			res.send("Create new category");
```
Comment (AudriusBaltusis) [inline controllers/categoriesControl.js]: You should use DB for all routes

File: routes/expenses.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const router = express.Router();
+const controller = require("../controllers/expensesControl");
+
+router.get("/", controller.getExpense);
```
Comment (AudriusBaltusis) [inline routes/expenses.js]: Authorization is not required for these routes?

**Comment by AudriusBaltusis** [comment]
Postman collection is missing

**Review by AudriusBaltusis — CHANGES_REQUESTED** [review]

**Comment by javdad** [comment]
> Postman collection is missing

I have Postman collection. Should I share the link?

**Comment by AudriusBaltusis** [comment]
> > Postman collection is missing
> 
> I have Postman collection. Should I share the link?

you have to commit it to repository

**Review by AudriusBaltusis — APPROVED** [review]

**Pull request merged**